src/helm/proxy/models.py

```diff
@@ -568,6 +568,14 @@ def engine(self) -> str:
         description="GPT-JT (6B parameters) is a fork of GPT-J",
         tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, GPTJ_TOKENIZER_TAG],
     ),
+    Model(
+        group="together",
+        creator_organization="Together",
+        name="together/gpt-neo-xt-chat-base-20b",
```

Comment:
Should also add a similar entry in `schema.yaml` (@yifanmai: people always forget to do this, so maybe we should put it in the comments here or instructions somewhere to remind people).